### PR TITLE
Javascript javascript3 week1

### PR DESCRIPTION
**Pull request by RodrigoGuinsbergPinto**
This is my JS3 week1 homework.

### CODE REVIEW

File: javascript/javascript3/week1/myFile.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "motorcycles": {
```
Comment (zmzade) [inline javascript/javascript3/week1/myFile.json]: It is ok but as there are a list of subsequent data, it is better to use an array.
so `"motorcycles":[Honda: { }, Yamaha:{ }, Ducati:{ }]`.

File: javascript/javascript3/week1/weather.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    <title>JS3-week1 HW</title>
+  </head>
+  <body>
+    <h1>Get weather forecast</h1>
+
+    <div class="getInfo">
+      <input
+        id="cityInput"
+        type="text"
+        class=""
```
Comment (zmzade) [inline javascript/javascript3/week1/weather.html]: It is good to remove `class`, as it is empty.

File: javascript/javascript3/week1/giphy.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+// Giphy API Key: Wyd7tK4cooBne57SnMhnvk2t813zedju
+
+const btn = document.querySelector("#searchBtn");
+
+const getGif = () => {
+  const gifNumbInput = document.querySelector("#gifNumbInput").value;
```
Comment (zmzade) [inline javascript/javascript3/week1/giphy.js:6]: As html element has an `id`, it is better to use `getElementById` method for grabbing it.
`document.getElementById('gifNumbInput')`

File: javascript/javascript3/week1/giphy.html [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="giphy.css" />
+    <title>Giphy</title>
+  </head>
+
+  <body>
+    <h1>Giphy Searcher</h1>
+    <main class="input-area">
+      <input id="wordInput" type="text" class="" placeholder="Enter a word" />
```
Comment (zmzade) [inline javascript/javascript3/week1/giphy.html]: It is good to remove `class` here too.

**Review by zmzade — COMMENTED** [review]
Great job, well done 👍 .
I just add few points, however you did it really nice. Perfect :).

**Comment by RodrigoGuinsbergPinto** [comment]
Hi zmzade,
here are my files with suggested changes.
Thank you so much for taking the time to review my homework. 🤩

**Review by zmzade — APPROVED** [review]

**Pull request merged**